### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by ahangarha**
Better to get updated after merging #16 and making a new release so that we can add the version diff section at the end of the changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+- Add cleanup-old-images command [PR10](https://github.com/shakacode/heroku-to-control-plane/pull/10) by [Rafael Gomes](https://github.com/rafaelgomesxyz)
+- Add cleanup-stale-apps command [PR9](https://github.com/shakacode/heroku-to-control-plane/pull/9) by [Rafael Gomes](https://github.com/rafaelgomesxyz)
+
+### Fixed
+
+- Fixed issue with default gems for older Ruby versions [PR14](https://github.com/shakacode/heroku-to-control-plane/pull/14) by [Rafael Gomes](https://github.com/rafaelgomesxyz)
+
+## [0.1.0] - 2023-02-19
+
+### Added
```
Comment (justin808) [inline CHANGELOG.md:27]: You are missing the link definitions at the bottom of the file. See other changelogs (view raw).

**Review by justin808 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ahangarha) [reply CHANGELOG.md:27]: Exactly.

At the moment of making this PR, we didn't have any tag on the project so I couldn't add version diff links at the bottom of the page. Now we have and I can add.

**Comment by justin808** [comment]
we can start the proper changelog with v1.0

**Comment by ahangarha** [comment]
@justin808 At least based on the stage of development this project is in, I think it makes sense to start making changelog from the first stable release (which should be version 1)

**Comment by justin808** [comment]
@rafaelgomesxyz can you take over this PR or close it and open a new one. We need to keep the changelog corrext.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,40 @@
-# Changelog
+# Change Log
```
Comment (ahangarha) [inline CHANGELOG.md:1]: I think it is better to keep it as `Changelog`.

https://keepachangelog.com/en/1.1.0/

**Comment by rafaelgomesxyz** [comment]
> @rafaelgomesxyz can you take over this PR or close it and open a new one. We need to keep the changelog corrext.

Done.

**Pull request merged**